exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -202,12 +218,14 @@ export default ExpFrameBaseComponent.extend({
       target.unshiftObject(card);
     },
     nextPage() {
+      this.set('cardSortResponse', Ember.copy(this.buckets, true));
```

Comment:
Hmm. It's possible that computed properties work weirdly with such deeply nested items. Long term, we may want to change how buckets are implemented.

One way around this for now: consider making `responses` volatile. Instead of watching a variable, it'll simply be recalculated every time you ask for it.